apps/client/src/components/Layouts/DefaultLayout.tsx

```diff
@@ -7,9 +7,11 @@ function DefaultLayout(props: { heading: string; children: React.ReactNode }) {
   const { location } = useHistory();
   const { trackPageView } = useMatomo();
 
-  React.useEffect(() => {
-    // @datapunt Track Page View https://github.com/Amsterdam/matomo-tracker/tree/master/packages/react
-    trackPageView({});
+  useEffect(() => {
+    if (!localStorage.getItem("doNotTrack")) {
+      // @datapunt Track Page View https://github.com/Amsterdam/matomo-tracker/tree/master/packages/react
+      trackPageView({});
+    }
```

Comment:
Duplicate code

Reply:
The only place trackPageView is called is here. 
So if we move it, it should be a different hoc. Otherwise pageviews are called multiple times. 
I think this is the clearest way.

Reply:
`MatomoPageView` can be new function inside the same hoc, right? And then only call inside `DefaultLayout.tsx`

![image](https://user-images.githubusercontent.com/7781865/85017420-b8e73a80-b16b-11ea-9145-4c5b9eb56005.png)